src/sentry/integrations/slack/endpoints/action.py

```diff
@@ -361,19 +356,66 @@ def post(self, request: Request) -> Response:
         attachment = SlackIssuesMessageBuilder(
             group, identity=identity, actions=action_list
         ).build()
-        body = self.construct_reply(attachment, is_message=self.is_message(data))
+        body = self.construct_reply(attachment, is_message=_is_message(slack_request.data))
 
         return self.respond(body)
 
-    def handle_member_approval(self, slack_request: SlackActionRequest) -> Response:
+    def handle_unfurl(self, slack_request: SlackActionRequest, action_option: str) -> Response:
+        organizations = slack_request.integration.organizations.all()
+        analytics.record(
+            "integrations.slack.chart_unfurl_action",
+            organization_id=organizations[0].id,
```

Comment:
@mgaeta we can't assume that the organization is the first one here, it may not be correct if that integration is used by multiple orgs. This is why should be putting the organization information into `callback_data`.

Reply:
This refactor keeps the previous behavior. I can fix this in another PR.